include/cutlass/gemm/collective/intel_pvc_mma.hpp

```diff
@@ -98,24 +100,44 @@ struct CollectiveMma<
   using TransformB = TransformB_;
   using ArchTag = typename DispatchPolicy::ArchTag;
 
-  static constexpr int SubgroupSize = DispatchPolicy::SubgroupSize;
-
   using MmaAtomShape = typename TiledMma::AtomShape_MNK;
   using SubgroupTileShape = decltype(tile_shape(TiledMma()));
+  WorkgroupTileShape wg_tile_shape;
+  SubgroupTileShape sg_tile_shape;
 
+  static constexpr auto wg_tile_m = decltype(get<0>(wg_tile_shape))::value;
+  static constexpr auto wg_tile_n = decltype(get<1>(wg_tile_shape))::value;
```

Comment:
Are these wg_tile_m and wg_tile_n the tile Size of the output matrix which is computed by one entire workgroup ?

Reply:
Wg_tile_m/n is the information about one entire workgroup, no matter matA/B/C. 
By the way, wg_tile_k usually equal matK.